api/groups/baseAccountsGroup_test.go

```diff
@@ -64,6 +64,26 @@ type getShardResponse struct {
 	Data getShardResponseData
 }
 
+type guardianDataApiResponse struct {
+	GeneralResponse
+	Data guardianDataApiResponseData `json:"data"`
+}
+
+type guardianDataApiResponseData struct {
+	GuardianData guardianData `json:"guardianData"`
+}
+
+type guardianData struct {
+	ActiveGuardian  guardian `json:"activeGuardian,omitempty"`
+	PendingGuardian guardian `json:"pendingGuardian,omitempty"`
+	Frozen          bool     `json:"frozen,omitempty"`
+}
+
+type guardian struct {
+	Address string `json:"address"`
+	Epoch   uint32 `json:"epoch"`
```

Comment:
Just a preference: I would have called this `ActivationEpoch`. Not necessarily in this test file.

Reply:
Yes, it is clearer this way, and this is also what we store the guardian data in the account.
Changed